bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/digitalinvoice/view/DigitalInvoiceOnboardingIllustrationAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package net.gini.android.bank.sdk.capture.digitalinvoice.view
+
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.core.content.ContextCompat
+import net.gini.android.capture.view.InjectedViewAdapter
+
+interface DigitalInvoiceOnboardingIllustrationAdapter: InjectedViewAdapter {
```

Comment:
[Suggestion] I see our `OnboardingIllustrationAdapter` in Capture SDK is generic enough to be used in the Bank SDK as well. 

What about making it public and reusing it? Then we don't need a dedicated one for digital invoice.